### PR TITLE
Adds example to documentation

### PR DESCRIPTION
**Pull request by nilssass**

### CODE REVIEW

File: src/particle-analysis/Histogram.py [hunk 1]
```diff
@@ -69,25 +69,59 @@ class Histogram:
...
     write_to_file:
         Write one histogram to a csv file.
+        
+    Examples
+    --------
+    **Creating histograms**
+    
+    To create multiple histograms with the same number and size of bins we can 
+    initialize the Histogram object with a tuple (otherwise initialize with a 
+    list containing all bin edges). To fill these histogramscstore them in a
```
Comment (Hendrik1704) [inline src/particle-analysis/Histogram.py:98]: "histograms store"

**Review by Hendrik1704 — CHANGES_REQUESTED** [review]
Looks good except for the one spelling mistake. I can not change it from my tablet.

File: src/particle-analysis/Histogram.py (hunk 1 above)
Comment (nilssass) [reply src/particle-analysis/Histogram.py]: Ahh, thanks... Corrected!

**Review by Hendrik1704 — APPROVED** [review]
Ok

**Pull request merged**